src/indieweb_utils/replies/context.py

```diff
@@ -10,14 +11,24 @@
 
 @dataclass
 class ReplyContext:
-    author_url: str
+    webmention_endpoint: str
+    post_url: str
+    photo: str
+    name: str
+    video: str
+    post_html: str
+    post_text: str
     author_name: str
```

Comment:
Could an article have multiple authors? I think we should split author into its own dataclass, then we can use it in our reply context. 

```
authors: List[PostAuthor]
```

If we're unable to find an author, this would just be an empty list, so it would be easy to use as a user of the library, too, just literate over it.